menas-web/ui/index.html

```diff
@@ -0,0 +1,114 @@
+<!--
+  ~ Copyright 2018 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE HTML>
+
+<html>
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta http-equiv='Content-Type' content='text/html;charset=UTF-8'/>
+
+    <link rel="icon" href="favicon.ico" type="image/x-icon" />
+
+    <script src="resources/sap-ui-core.js" id="sap-ui-bootstrap"
+            data-sap-ui-compatVersion="edge"
+            data-sap-ui-preload="async"
+            data-sap-ui-libs="sap.m,sap.ui.table,sap.ui.layout,sap.tnt"
+            data-sap-ui-bindingSyntax="complex"
+            data-sap-ui-theme='sap_belize'></script>
+
+    <script src="node_modules/moment/min/moment.min.js"></script>
+    <script src="node_modules/lodash/lodash.js"></script>
+    <script src="node_modules/cronstrue/dist/cronstrue.js"></script>
+
+    <script>
+        function _menasLoadElem(sElemType, sSrc, aAttrs, fnOnLoad) {
+            const el = document.createElement(sElemType);
+            const srcAttr = sElemType === "link" ? "href" : "src";
+            el[srcAttr] = `${sSrc}`;
+            aAttrs.forEach((attr) => {
+                el.setAttribute(attr.k, attr.v);
+            })
+            if (fnOnLoad) {
+                el.addEventListener("load", fnOnLoad);
+            }
+            document.getElementsByTagName("head").item(0).append(el);
+        }
+
+        $.getJSON('package.json', function(json) {
+            window.apiUrl = json.config.apiUrl + "/api/";
```

Comment:
URLs should not have trailing `/`

```suggestion
            window.apiUrl = json.config.apiUrl + "/api";
```

This is a general standard to make sure that everyone can contact paths in URL without thinking about it.

Reply:
I was more on the thinking to add the trailing only once in /api/, so that anywhere else it wouldn't be needed

Reply:
- `/api/` is a 2 segment URI, with api and empty as the segments. This immediately stinks
- When building URIs (concatenating them), look at spring, you are expected to start with a `/` but not have any at the end. This in my experience is a rule followed by people regardless of platform/language

Please remove the trailing / from your URIs.